test/migrators/module/indirect_configuration.hrx

```diff
@@ -0,0 +1,22 @@
+<==> arguments
+--migrate-deps
+
+<==> input/entrypoint.scss
+$config: red;
+@import "direct";
+
+<==> input/_direct.scss
+@import "indirect";
+
+<==> input/_indirect.scss
+$config: green !default;
+
+<==> output/entrypoint.scss
+$config: red;
+@use "direct" with (
+  $config: red
+);
```

Comment:
It would be nice to generate this as `@use "direct" with ($config: red);` for the case where only a single variable is being configured.